### DISCUSSION TITLE
I actually like hacking the docs

### DISCUSSION THREAD

**Discussion by stepheweffie** [zauberzeug/nicegui#2176] [General]
Hi,
I just wanted to start a silly optimist thread about how to hack the docs. I think the documentation is pretty good and has some imposed "fun" challenges to it that require looking at the examples code, trying and figuring stuff out from scratch, picking up skills on-the-fly, and I really like this framework. 

I'm building a blog CMS for containers and I think I have seen everything I need to figure it out in the project already.

**Comment by rodja** [comment]
What do you mean with the title "I actually like hacking the docs"? I read it like "I really like the docs and they bring me easily up to speed". Is that correct?

**Reply by stepheweffie** [reply]
I wouldn't say it's totally straight forward, but it's compelling to read them.
